### ISSUE TITLE
Proposed Timeline

### ISSUE THREAD

**Issue by TuffDev** [EPICMC/CTF-SERVER#2]
Jan 19 - 20
Implement sign refreshing classes
Start plugin frame

Jan 21
Create event listener framework and code outline

Jan 22
Add event listener class

Jan 23
Add main class, player management.

Jan 23 -24 
Implement fake player spawn??

Jan 24 -25
Debugging

Jan 25
Projected beta release

Jan 35 
Projected Alpha release

**Comment by rj0495** [comment]
Jan 35???

**Comment by rj0495** [comment]
Also, your back?

**Comment by TuffDev** [comment]
Yeah, hey commit the code that you have.  You said you had a fair bit done.

**Comment by daltontastic** [comment]
Excellent work getting organized. You guys are awesome! :+1:
